api/specs/webserver/v0/openapi-auth.yaml

```diff
@@ -0,0 +1,172 @@
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/DefaultErrorResponse'
+
+  /auth/login:
+    post:
+      operationId: auth_login
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/login.yaml#/LoginFormType'
+      responses:
+        '200':
+          description: Succesfully logged in
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
+        '401':
+          $ref: './openapi-auth.yaml#/components/responses/AuthError_Unauthorized_401'
+        '422':
+            $ref: './openapi-auth.yaml#/components/responses/DataError_UnprocessableEntity_422'
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/DefaultErrorResponse'
+
+  /auth/logout:
+    get:
+      operationId: auth_logout
+      responses:
+        '200':
+          description: Succesfully logged out
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/DefaultErrorResponse'
+
+  #TODO: /auth/reset-password:
+
+  /auth/change-email:
+    post:
+      operationId: auth_change_email
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                new_email:
+                  type: string
+                  #format: email
+      responses:
+        '200':
+          description: User has been succesfully registered.
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/DefaultErrorResponse'
+
+  #TODO: /auth/change-password:
+
+  /auth/confirmation/{code}:
+    get:
+      operationId: auth_confirmation
+      parameters:
+        - name: code
+          in: path
+          required: true
+          schema:
+            type: string
+            #format: uuid
+      responses:
+        default:
+          $ref: './openapi-auth.yaml#/components/responses/OK_NoContent_204'
+
+components:
+  responses:
+    # TODO:Envelope objects are still not well/easily defined. See discriminators
+    OK_NoContent_204:
+      description: everything is OK, but there is no content to return
+
+    DefaultErrorResponse:
+      description: Unexpected error
+      content:
+        application/json:
+          schema:
+            $ref: './components/schemas/error.yaml#/ErrorEnveloped'
+
+    DataError_BadRequest_400:
+      description: requested information is incomplete or malformed
+      content:
+        application/json:
+          schema:
+            $ref: './components/schemas/error.yaml#/ErrorEnveloped'
```

Comment:
these errors all contain the same object. Why not then use the same ? The numbers in the responses are already clear enough no?